cli/static/generate.go

```diff
@@ -0,0 +1,22 @@
+// +build ignore
+
+package main
+
+import (
+	log "github.com/sirupsen/logrus"
+
+	"github.com/linkerd/linkerd2/cli/static"
+	"github.com/shurcooL/vfsgen"
```

Comment:
If you remove the newline in the imports here, `gofmt` will sort the logrus import between the static and vfsgen imports.